src/routes/docs/builders/separator/preview.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { Docs } from '$routes/(components)';
+	import { createSeparator } from '@melt-ui/svelte/builders/separator';

```

Comment:
Should be $lib. Also, don't forget to export it from '$lib/builders/index.ts' as well